packets/packets.go

```diff
@@ -181,8 +181,9 @@ type Subscription struct {
 
 // InlineSubscription represents an internal subscription, and the Handler is a callback function used to process messages.
 type InlineSubscription struct {
-	Filter  string
-	Handler func(filter string, packet Packet)
+	Filter     string
+	Identifier string
```

Comment:
This can be confused with Subscription Identifiers, which are integers. I recommended renaming to `ClientID string` to be less specific. I did consider just `ID`, but this is still ambiguous in some of the later calls and params.

```suggestion
	ClientID string
```